### ISSUE TITLE
Autocomplete shows up after typing a certain characters

### ISSUE THREAD

**Issue by Stardidi** [jacknbeans/hexe#5]
When you type a space on an empty line the auto complete window will show up. Same with any operator (+=*^%$#@!~), however when typing an operator the autocomplete will disappear if you then type a space.

This is slightly annoying behavior, as you have autocomplete in your face at moments where you don't want it. 

A way to improve this might be to make it only show up if you have typed the beginning of a recognized string or when you press a certain keyboard shortcut.

**Comment by jacknbeans** [comment]
I've now changed it so that the autocomplete list only appears when a familiar character in the scriptbinds json file is used. However, I haven't made it possible to show the list with a keyboard shortcut. I will spend some more time thinking of how that would be possible. 

Commit https://github.com/jacknbeans/hexe/commit/19b06622c51d9ca96b3bc82ceac312f75705e150 made this change and has also been published to atom.

**Comment by jacknbeans** [comment]
Actually, leaving this issue open for now till I might be able to find the keyboard shortcut for it.

**Comment by jacknbeans** [comment]
Turns out, was a lot easier than I thought. Autocomplete plus (which comes packaged with atom already) has a keyboard shortcut for triggering the autocomplete list manually. So I've added a check for if this happens and it will show the list of script binds if no prefix is present.

So now it won't be triggered when a space or any other character is typed only if a character is recognized. And if one wants to see the list before typing anything, pressing ctrl-space (default shortcut) will show the list of script binds.

Commit https://github.com/jacknbeans/hexe/commit/e0e5a50bd6a40d22150f063f7678a8084e326b4a added this functionality.
